common/BUILD

```diff
@@ -41,28 +35,17 @@ java_library(
     tags = ["maven_coordinates=grakn.core:common:{pom_version}"],
```

Comment:
Let's move `runtime_deps` to the actual package that needs it. In this case: `//common/config`